### ISSUE TITLE
Can I use with Python 3?

### ISSUE THREAD

**Issue by HigorSilvaRosa** [shymonk/django-datatable#66]
A think that this package is awesome, but this is only on Python 2. There are plans to port it to Python 3? How can I help?

**Comment by shymonk** [comment]
Yeah, i will port it to Python 3 based on https://github.com/shymonk/django-datatable/pull/30/files,
But, it needs a few weeks, i am not familiar with Python 3. May be we can do this task together.

**Comment by HigorSilvaRosa** [comment]
Hi!
Sorry... Do you still need help?

I'm here, now!

**Comment by 18926589010** [comment]
i am using this packets on my project env python3.6+ django2. it works fine
